src/main/java/com/amplitude/HttpCall.java

```diff
@@ -94,7 +97,7 @@ protected Response syncHttpCallWithEventsBuffer(List<Event> events) {
 
         }
       }
-      return responseBody;
     }
+    return responseBody;
```

Comment:
I see, so in this case, if the exception is thrown and not caught, nothing will be returned.